test/over_react/shared/map_proxy_tests.dart

```diff
@@ -50,12 +50,12 @@ void mapProxyTests(Map mapProxyFactory(Map proxiedMap)) {
       var testMap = {};
       when(mock.addAll(any)).thenReturn('value');
 
-      expect(proxy.addAll(testMap), isNull);
+      expect(() => proxy.addAll(testMap), isNull);
```

Comment:
Huh, you can pass in a function and `expect` will evaluate it and match the result? Is this documented somewhere?

Reply:
That I don't know for sure @greglittlefield-wf ... I did this to address a dart 2 analyzer error, and the test didn't fail.  I didn't dig too much further into it.

Reply:
In addition, these test are currently not run in the DDC because they us mirrors.

Reply:
It looks like it doesn't work. This is causing failures in Dart 1:
```
00:16 +245 -20: [Chrome] test/over_react_component_declaration_test.dart: component base: UiState provides Map functionality: proxies the Map member: forEach [E]
  Expected: null
    Actual: <Closure: () => void from: () => proxy[$forEach](callback)>
  
  package:test                                 expect
  over_react/shared/map_proxy_tests.dart 86:7  test.test.dart.fn
```

We'll have to find a different workaround for this issue; perhaps we just shouldn't assert the return value. Or just comment out the tests until we're running them in Dart 2.

Reply:
@greglittlefield-wf this branch is targeting an SDK version of 2.0.0-dev.49.0, I wouldn't expect a lot of tests in this branch to pass using the dart 1 SDK.